src/django/api/libs.py

```diff
@@ -0,0 +1,44 @@
+from django.conf import settings
```

Comment:
Why `libs.py` instead of something like `geocoding.py`?

Reply:
No real reason, although `geocoding.py` is probably a better name. Will update!

Reply:
I updated this and force pushed an amended commit.